packages/backend/app/pkg/kvs/client.go

```diff
@@ -0,0 +1,63 @@
+package kvs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/redis/go-redis/v9"
+	"github.com/stlatica/stlatica/packages/backend/app/pkg/logger"
+)
+
+// Client is an interface for key-value store.
+type Client interface {
+	// SetValue sets a key value pair in valkey.
+	SetValue(ctx context.Context, key string, value string) error
+	// GetValue retrieves a value for a given key from valkey.
+	GetValue(ctx context.Context, key string) (string, error)
+}
+
+type client struct {
+	client    *redis.Client
```

Comment:
redis.Clientはinterfaceなのでpointer指定する必要はありません

合わせて、struct nameと混在してしまいそうなのでorgClientなど名称を変更していただけますか？ :pray:

```suggestion
	orgClient redis.Client
```